### DISCUSSION TITLE
Portal server

### DISCUSSION THREAD

**Discussion by og1-alarm** [serverpod/serverpod#2026] [General]
Hey Serverpod team.

I am interested to know if there is any possibility to develop a portal app using serverpod_auth_server.
The basic idea is to have a portal app, that allows you to register + log in and open other apps (that require the same login so to keep the session for them).

As well separate app that the user can install and register + log in to it even without the portal app (but to register on the portal server).
This will require the portal server to have its own db, while the other apps to access the portal db for registration and their own db for their own stuff.

If this is possible, could you please suggest an architecture approach on how to accomplish this using flutter + serverpod for all the apps?

If I should clarify something please let me know.
Thank you!

**Comment by SandPod** [comment]
Hey @og1-alarm!

Since this is not related to a bug or a feature I will convert this to a discussion post.
That might also increase the chances of getting architecture advice from the community as well.

**Comment by og1-alarm** [comment]
Thanks @SandPod , really curious to see if our business scenario is supported by Serverpod!

**Comment by og1-alarm** [comment]
@vlidholt could you please give your opinion on this architectural approach.
Is it currently supported or possibly in the future?
